tables/ro.ctb

```diff
@@ -1,126 +1,158 @@
-uppercase B 127		LATIN CAPITAL LETTER B
-uppercase C 147		LATIN CAPITAL LETTER C
-uppercase D 1457		LATIN CAPITAL LETTER D
-uppercase E 157		LATIN CAPITAL LETTER E
-uppercase F 1247		LATIN CAPITAL LETTER F
-uppercase G 12457		LATIN CAPITAL LETTER G
-uppercase H 1257		LATIN CAPITAL LETTER H
-uppercase I 247		LATIN CAPITAL LETTER I
-uppercase J 2457		LATIN CAPITAL LETTER J
-uppercase K 137		LATIN CAPITAL LETTER K
-uppercase L 1237		LATIN CAPITAL LETTER L
-uppercase M 1347		LATIN CAPITAL LETTER M
-uppercase N 13457		LATIN CAPITAL LETTER N
-uppercase O 1357		LATIN CAPITAL LETTER O
-uppercase P 12347		LATIN CAPITAL LETTER P
-uppercase Q 123457		LATIN CAPITAL LETTER Q
-uppercase R 12357		LATIN CAPITAL LETTER R
-uppercase S 2347		LATIN CAPITAL LETTER S
-uppercase T 23457		LATIN CAPITAL LETTER T
-uppercase U 1367		LATIN CAPITAL LETTER U
-uppercase V 12367		LATIN CAPITAL LETTER V
-uppercase W 24567		LATIN CAPITAL LETTER W
-uppercase X 13467		LATIN CAPITAL LETTER X
-uppercase Y 134567		LATIN CAPITAL LETTER Y
-uppercase Z 13567		LATIN CAPITAL LETTER Z
-lowercase a 1		LATIN SMALL LETTER A
-lowercase b 12		LATIN SMALL LETTER B
-lowercase c 14		LATIN SMALL LETTER C
-lowercase d 145		LATIN SMALL LETTER D
-lowercase e 15		LATIN SMALL LETTER E
-lowercase f 124		LATIN SMALL LETTER F
-lowercase g 1245		LATIN SMALL LETTER G
-lowercase h 125		LATIN SMALL LETTER H
-lowercase i 24		LATIN SMALL LETTER I
-lowercase j 245		LATIN SMALL LETTER J
-lowercase k 13		LATIN SMALL LETTER K
-lowercase l 123		LATIN SMALL LETTER L
-lowercase m 134		LATIN SMALL LETTER M
-lowercase n 1345		LATIN SMALL LETTER N
-lowercase o 135		LATIN SMALL LETTER O
-lowercase p 1234		LATIN SMALL LETTER P
-lowercase q 12345		LATIN SMALL LETTER Q
-lowercase r 1235		LATIN SMALL LETTER R
-lowercase s 234		LATIN SMALL LETTER S
-lowercase t 2345		LATIN SMALL LETTER T
-lowercase u 136		LATIN SMALL LETTER U
-lowercase v 1236		LATIN SMALL LETTER V
-lowercase w 2456		LATIN SMALL LETTER W
-lowercase x 1346		LATIN SMALL LETTER X
-lowercase y 13456		LATIN SMALL LETTER Y
-lowercase z 1356		LATIN SMALL LETTER Z
-punctuation \x00a0 0		NO-BREAK SPACE
-uppercase \x00c2 1267		LATIN CAPITAL LETTER A WITH CIRCUMFLEX
-uppercase \x00ce 1467		LATIN CAPITAL LETTER I WITH CIRCUMFLEX
-lowercase \x00e2 126		LATIN SMALL LETTER A WITH CIRCUMFLEX
-lowercase \x00ee 146		LATIN SMALL LETTER I WITH CIRCUMFLEX
-uppercase \x0102 167		LATIN CAPITAL LETTER A WITH BREVE
-lowercase \x0103 16		LATIN SMALL LETTER A WITH BREVE
-uppercase \x015e 1567		LATIN CAPITAL LETTER S WITH CEDILLA
-lowercase \x015f 156		LATIN SMALL LETTER S WITH CEDILLA
-uppercase \x0162 2467		LATIN CAPITAL LETTER T WITH CEDILLA
-lowercase \x0163 246		LATIN SMALL LETTER T WITH CEDILLA
-uppercase \x0218 1567		LATIN CAPITAL LETTER S WITH COMMA BELOW
-lowercase \x0219 156		LATIN CAPITAL LETTER S WITH COMMA BELOW
-uppercase \x021a 2467		LATIN CAPITAL LETTER T WITH COMMA BELOW
-lowercase \x021b 246		LATIN SMALL LETTER T WITH COMMA BELLOW
+#

```

Comment:
I guess there must have been some change in the line endings of this file or something, because Github says a lot of lines changed, but I can only see a few actual changes. @jkinslayer Could you fix that?